.github/workflows/build.yml

```
@@ -0,0 +1,18 @@
+name: build the package
+on:
+  pull_request:
+jobs:
+  run:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: INstall Go
+        uses: actions/setup-go@v2
```

```suggestion
        uses: actions/setup-go@v3
```

v3 is released: https://github.com/marketplace/actions/setup-go-environment#v3 , could we use that?

---

updated